ngraph_bridge/ngraph_deassign_clusters.cc

```diff
@@ -51,7 +51,7 @@ namespace ngraph_bridge {
 // NGRAPH_TF_DISABLE_DEASSIGN_CLUSTERS=1.
 //
 
-static const int MIN_NONTRIVIAL_NODES = 2;
+static const int MIN_NONTRIVIAL_NODES = 6;
```

Comment:
Curious, how did you decide with the number? Is it related to the model you were running?

Reply:
Not really. Generally speaking, we'd need some sort of symbolic analysis/execution coupled with a cost model to come up with a good heuristic for cluster triviality. Nevertheless, 2 seems like a really low number for a trivial cluster. Given our current op coverage and the average number of nodes/cluster for the standard set of models we test against, 6 seems like a safe minimum?